### ISSUE TITLE
Check email link text color

### ISSUE THREAD

**Issue by jgarber623-gov** [Code-dot-mil/code.mil#234]
It's light grey for some reason! 🤷‍♂️ 

It should be white.

<img width="518" alt="screen shot 2018-07-10 at 2 57 04 pm" src="https://user-images.githubusercontent.com/27780860/42531278-96eb1904-8451-11e8-8ad9-3dd849b82b62.png">

**Comment by josephwhittington** [comment]
I will be working in this issue

**Comment by milovanderlinden** [comment]
Add

```
.usa-section-dark a {
  color: #fff !important;
}
```

to custom.css and you are good to go.

**Comment by jordangov** [comment]
Hmm... we could... but using `!important` is not a good approach generally in CSS. It means if we ever need to overwrite that style we'd have to use yet another `!important` later in the css. I'd rather find a solution to _why_ that button is gray and the other is white.

**Comment by milovanderlinden** [comment]
That button is gray because it is defined like that in the uswds `main.css`

**Comment by jordangov** [comment]
But why is the other button not gray? In any case, we can override the styles without using `!important`, it's really a nuclear option that should be avoided.

**Comment by jgarber623-gov** [comment]
Dug into this for a minute and, despite Chrome Inspector's ordering of applied CSS rules, it appears that this rule is the dominant:

```
.usa-section-dark a {
  color: #d6d7d9;
}
```

That rule appears toward the bottom of `main.css` and has the same or slightly higher specificity as other rules applying color to this element. 🙃

**Comment by jgarber623-gov** [comment]
Fixed in a39b566 by adding:

```yaml
class: page-home
```

...to the `index.md` file and the following to `custom.css`:

```css
.page-home .usa-section-dark .usa-button {
  color: #fff;
}
```

🇺🇸
